73063.typ

```diff
@@ -177,6 +177,15 @@
 		([Touchdown], [SLIGHTLY TAIL LOW]),
 		([Brakes], [APPLY HEAVILY]),
 	)
+	// From the GFC 500 AFMS.
+	#checklist("Autopilot Malfunction", palette.brown, emergency: true,
+		([Control wheel], [GRIP FIRMLY]),
+		([AP DISC button], [PRESS AND HOLD]),
+		([Aircraft attitude], [MAINTAIN]),
+		([Trim], [RE-TRIM]),
+		([Autopilot circuit breaker], [PULL]),
+		([AP DISC button], [RELEASE]),
+	)
```

Comment:
The emergency checklists from Cessna's POH are in the same order as the POH, but this checklist is from a different document so I had to find a place to put it. I decided to put it in the leftmost column, because that column had the most space already, and it wasn't splitting up a group of related checklists.